### PR TITLE
Nityaravi quickstart build

### PR DESCRIPTION
**Pull request by nityaravi**

### CODE REVIEW

**Review by navierula — APPROVED** [review]

**Review by gtothe7 — COMMENTED** [review]
- [ ]

File: quickstart-build/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# quickstart-build
```
Comment (gtothe7) [inline quickstart-build/README.md:1]:
```suggestion
# quickstart-build
```

Comment (gtothe7) [inline quickstart-build/README.md:1]: - 
```suggestion
- # quickstart-build****
```

**Review by gtothe7 — COMMENTED** [review]
- [ ]

File: quickstart-build/README.md (hunk 1 above)
Comment (gtothe7) [inline quickstart-build/README.md:1]: [1
```suggestion
# quickstart-build
```
](****)

**Comment by gtothe7** [comment]
[build]()

**Review by gtothe7 — COMMENTED** [review]
- - [complete****]

File: quickstart-build/README.md (hunk 1 above)
Comment (gtothe7) [inline quickstart-build/README.md:1]:
```suggestion
# quickstart-build
```

File: quickstart-build/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM alpine
+COPY quickstart.sh /
+CMD ["/quickstart.sh"]
```
Comment (gtothe7) [inline quickstart-build/Dockerfile:3]:
```suggestion
CMD ["/quickstart.sh"]
```

File: quickstart-build/cloudbuild.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
```
Comment (gtothe7) [inline quickstart-build/cloudbuild.yaml:2]:
```suggestion
- - name: 'gcr.io/cloud-builders/docker' 
```

File: quickstart-build/quickstart.sh [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```
Comment (gtothe7) [inline quickstart-build/quickstart.sh:1]:
```suggestion
- #!/bin/sh
```

**Review by gtothe7 — COMMENTED** [review]
- - [ [complete 
]()] - - [**** ]

**Pull request merged**